src/main/resources/view/OrderListPanel.fxml

```diff
@@ -3,6 +3,6 @@
 <?import javafx.scene.control.ListView?>
 <?import javafx.scene.layout.VBox?>
 
-<VBox xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<VBox xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Should this be changed back to 8? PersonListPanel.fxml uses `http://javafx.com/javafx/8` still. Should be standardized throughout our fxml files.

Reply:
Good catch, Ill update this